sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -11,7 +11,7 @@ azure-arm: true
 generate-model-factory: false
 library-name: Compute
 namespace: Azure.ResourceManager.Compute
-require: https://github.com/Azure/azure-rest-api-specs/blob/03261080b3083a9e8cb0b61d840cc8291c596590/specification/compute/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/b06a6f45e472dd07c2e0dab06b1e52dfe2684c88/specification/compute/resource-manager/readme.md
```

Comment:
Do you want to wait with other updates or release a new package for this change?
Additionally, kindly update the changelog file in accordance.

Reply:
Please also note the release date in the changelog file if you want to release a new package for this change.

Reply:
> Do you want to wait with other updates or release a new package for this change? Additionally, kindly update the changelog file in accordance.

yes. we will wait for the community gallery fix PR https://github.com/Azure/azure-sdk-for-net/pull/36025/files. I've put the 'do not merge' tag on this one. thank you

Reply:
Do I need to add the new version header and date and everything else to changelog file? @ArthurMa1978 @ArcturusZhang

Reply:
Not this time, but merge this pr first

Reply:
> Do I need to add the new version header and date and everything else to changelog file? @ArthurMa1978 @ArcturusZhang

Add changes to current 1.2.0 changelog

Reply:
updated

Reply:
thank you @ArthurMa1978